generators/python-v2/base/src/asIs/core.ts

```diff
@@ -0,0 +1,6 @@
+import { python } from "@fern-api/python-ast";
+
+export const serialize_datetime = python.reference({
```

Comment:
I'd consider moving this into a separate directory - the `asIs` directory is usually used for all the files copied directly into the SDK (e.g. [C#](https://github.com/fern-api/fern/tree/main/generators/csharp/codegen/src/asIs), [PHP](https://github.com/fern-api/fern/tree/main/generators/php/codegen/src/asIs)), whereas this is used as a library for model generation (e.g. ObjectGenerator).

Reply:
Makes sense (will make this change in a FLUP)